package.json

```
@@ -1,6 +1,6 @@
 {
   "name": "stylelint",
-  "version": "14.9.1",
+  "version": "14.10.0",
```

[suggest] Could you revert the version, please? It will be automatically updated on release.

```suggestion
  "version": "14.9.1",
```